ubuntu.json

```diff
@@ -155,8 +155,13 @@
     "key_url": "https://dl-ssl.google.com/linux/linux_signing_key.pub"
   },
   {
-    "alias": "google-cloud",
-    "sourceline": "deb http://packages.cloud.google.com/apt trusty main",
+    "alias": "google-cloud-sdk-precise",
+    "sourceline": "deb http://packages.cloud.google.com/apt cloud-sdk-precise main",
+    "key_url": "https://packages.cloud.google.com/apt/doc/apt-key.gpg"
+  },
+  {
+    "alias": "google-cloud-sdk-trusty",
```

Comment:
Changing the alias will break existing builds, so please refrain from doing this.